src/topology/bounded_continuous_function.lean

```diff
@@ -672,6 +672,9 @@ instance : ring (α →ᵇ R) :=
   right_distrib := λ f₁ f₂ f₃, ext $ λ x, right_distrib _ _ _,
   .. bounded_continuous_function.add_comm_group }
 
+@[simp] lemma coe_mul (f g : α →ᵇ R) : ⇑(f * g) = λ x, (f x) * (g x) := rfl
```

Comment:
I think it might be better to avoid the lambda:
```suggestion
@[simp] lemma coe_mul (f g : α →ᵇ R) : ⇑(f * g) = ⇑f * ⇑g := rfl
```

Reply:
Agreed. I've just pushed https://github.com/leanprover-community/mathlib/pull/6867/commits/684ebbbc1a938279dd1f76a9f54ff054e3c97eda which does this and likewise for `coe_add` and `coe_sub`.